fabfile/pgbench_confs/pgbench_cloud.ini

```diff
@@ -26,10 +26,10 @@ sql_command: SELECT create_distributed_table('pgbench_tellers', 'tid');
 sql_command: VACUUM ANALYZE pgbench_tellers;
 
 [default-pgbench]
-pgbench_command: pgbench -c 32 -j 16 -T 300 -P 10 -r
+pgbench_command: pgbench -c 32 -j 16 -T 5 -P 10 -r
 
 [simple-update]
-pgbench_command: pgbench -c 32 -j 16 -T 300 -P 10 -r -b simple-update
+pgbench_command: pgbench -c 32 -j 16 -T 5 -P 10 -r -b simple-update
 
 [select-only]
-pgbench_command: pgbench -c 32 -j 16 -T 300 -P 10 -r -b select-only
+pgbench_command: pgbench -c 32 -j 16 -T 5 -P 10 -r -b select-only
```

Comment:
5 seconds?

Reply:
I have changed it to 5 seconds to test the whole structure fast. I have already added this to the pr description as `things to do before merging`